packages/cli/src/commands/account/get-payment-delegation.ts

```diff
@@ -0,0 +1,35 @@
+import BigNumber from 'bignumber.js'
+import { cli } from 'cli-ux'
+import { BaseCommand } from '../../base'
+import { Flags } from '../../utils/command'
+
+export default class GetPaymentDelegation extends BaseCommand {
+  static description =
+    "Get the payment delegation account beneficiary and fraction allocated from a validator's payment each epoch. The fraction is given as FixidityLib value and cannot be greater than 1."
```

Comment:
Since we report this as a human readable fraction, probably don't need the mention of FixidityLib.